src/init/pkg_installer.sh

```diff
@@ -149,6 +149,12 @@ mkdir -p ./backup
 
 echo "$(date +"%Y/%m/%d %H:%M:%S") - Generating Backup." >> ./logs/upgrade.log
 
+# Check tar dependency
+if ! which tar &> /dev/null; then
+    echo "$(date +"%Y/%m/%d %H:%M:%S") - The tar package is required for this operation, exiting." >> ./logs/upgrade.log
```

Comment:
It would be a good idea to output this error both in the upgrade.log and in the standard output so that the user sees this warning as soon as it occurs.

```suggestion
    echo "$(date +"%Y/%m/%d %H:%M:%S") - The tar package is required for this operation, exiting." >> ./logs/upgrade.log
```

Reply:
Applied with https://github.com/wazuh/wazuh/pull/21729/commits/297054eda5c69aa0c1d91e183f5efca4645fc861